remme/rpc_api/event/_handlers.py

```diff
@@ -134,14 +134,30 @@ class BlockEventHandler(BaseEventHandler):
     def hash_keys(cls):
         return ('id',)
 
-    def prepare_response(self, state, validated_data):
-        return {'id': state}
+    async def prepare_response(self, state, validated_data):
+        """
+        Prepare response.
+
+        To fetch block by number, consider blocks like elements in array. To get second, count from zero.
+        """
+        block_information_client = BlockInfoClient()
+        block_information = await block_information_client.get_block_info(block_num=int(state.get('block_num')) - 1)
+
+        block_identifier = state.get('block_id')
+
+        return {
+            'id': block_identifier,
+            'timestamp': block_information.timestamp,
+        }
 
     def parse_evt(self, evt):
-        try:
-            return next(filter(lambda el: el['key'] == 'block_id', evt['attributes']))['value']
-        except StopIteration:
-            pass
+        parsed_attributes = {}
+
+        for attribute in evt.get('attributes'):
+            if attribute.get('key') is not None:
+                parsed_attributes[attribute.get('key')] = attribute.get('value')
```

Comment:
`attribute['key']`

Reply:
@DeV1doR, why not my approach? I think if you can consider a code as like simple English language, you are on good way.

Reply:
@dmytrostriletskyi because u already check if `dict` has an element with 'key' key, why u need to call `.get` twice?

Reply:
@dmytrostriletskyi ICYMI I voted for it :)

Reply:
@eugene-babichenko, solved by separated variable.